meson.build

```diff
@@ -80,6 +80,8 @@ else
 	)
 endif
 
+patching = run_command('./patches/apply.sh', capture: true, check: true)
```

Comment:
Does this work as is? I thought `sh -c` was needed.

Reply:
It does work. The [meson docs](https://mesonbuild.com/External-commands.html) mention calling a bash script directly with run_command() so I tried it out